jbmc/regression/jbmc/validate-trace/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+TestClass.class
+--validate-trace --function TestClass.test
+^EXIT=10$
+^SIGNAL=0$
+Trace validation successful
```

Comment:
Please add a test for at least one error case.

Reply:
I don't know how to write a java program that would reliably have a trace that does not meet the criteria in the option --validate-trace. The unit tests cover the failure cases well.